chihiro_todo/main.js

```diff
@@ -165,7 +190,74 @@ document.addEventListener('DOMContentLoaded', function() {
         selectedPersonId.removeChild(deleteSelectPersonElm);
         personList = personList.filter(person => person.id !== personId);
     }
+    
+    const appendEditInput = (personId) => {
+        if (underEditedPersonId !== undefined) {
+            cancelEditPerson(underEditedPersonId);
+        }
+        underEditedPersonId = personId;
+        const person = personList.find(person => person.id === personId);
+        const beforeChangeElm = document.querySelector('#person_' + personId);
+        const personNameBeforeChengeElm = beforeChangeElm.querySelector('div');
```

Comment:
@chihiro-kakizaki 
[nits]
```suggestion
        const personNameBeforeChangeElm = beforeChangeElm.querySelector('div');
```
ここが綴り間違ってるみたいですね！

Reply:
@Shibata0806
気付きませんでした💦
ご指摘ありがとうございます🙇‍♀️
以下のコミットにて修正しました。
[personNameBeforeChangeElmのスペルミス修正](https://github.com/chihiro-kakizaki/todolist/pull/4/commits/3f87c479882b9ab8dc5c732b44327cfda0a30006)